docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,892 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Our encoded Hamiltonian is:\n",
+      "( SparsePauliOp(['XX', 'XY', 'XZ', 'YX', 'ZX', 'YY', 'YZ', 'ZY', 'ZZ'],\n",
+      "              coeffs=[1.5+0.j, 1.5+0.j, 1.5+0.j, 1.5+0.j, 1.5+0.j, 1.5+0.j, 1.5+0.j, 1.5+0.j,\n",
+      " 1.5+0.j]) ).\n",
+      "\n",
+      "We achieve a compression ratio of (6 binary variables : 2 qubits) ≈ 3.0.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qiskit_optimization.algorithms.qrao import QuantumRandomAccessEncoding\n",
+    "\n",
+    "\n",
+    "# Create an encoding object with a maximum of 3 variables per qubit, aka a (3,1,p)-QRAC\n",
+    "encoding = QuantumRandomAccessEncoding(max_vars_per_qubit=3)\n",
+    "\n",
+    "# Encode the QUBO problem into an encoded Hamiltonian\n",
+    "encoding.encode(problem)\n",
+    "\n",
+    "# This is our encoded Hamiltonian\n",
+    "print(f\"Our encoded Hamiltonian is:\\n( {encoding.qubit_op} ).\\n\")\n",
+    "print(\n",
+    "    \"We achieve a compression ratio of \"\n",
+    "    f\"({encoding.num_vars} binary variables : {encoding.num_qubits} qubits) \"\n",
+    "    f\"≈ {encoding.compression_ratio}.\\n\"\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Solve the problem using the `QuantumRandomAccessOptimizer`\n",
+    "\n",
+    "Having successfully encoded our input problem as a relaxed Hamiltonian, we proceed to solve it using the `QuantumRandomAccessOptimizer`. This optimizer allows us to find an approximate solution to the relaxed problem by leveraging quantum computing techniques.\n",
+    "\n",
+    "To set up the optimizer, we need to specify two crucial components:\n",
+    "\n",
+    "1. **Minimum Eigensolver**: We specify a minimum eigensolver to heuristically search for the ground state of the relaxed problem Hamiltonian. As an example, we can use the Variational Quantum Eigensolver (VQE). For simulation purposes, we'll employ an simulator, but you can choose a quantum device as the backend if desired.\n",
+    "2. **Rounding Scheme**: To map the ground state results back to a solution for the original problem, we specify a rounding scheme. By default, the `SemideterministicRounding` is used, but alternative scheme, `MagicRounding`, is also available."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit_algorithms.optimizers import COBYLA\n",
+    "from qiskit_algorithms.minimum_eigensolvers import VQE\n",
```

Comment:
```suggestion
    "from qiskit_algorithms import VQE\n",
```
This can now be imported directly as it used to be - in qiskit_algorithms this is the new primitive based VQE